### ISSUE TITLE
Duplicated and non-upstream resources

### ISSUE THREAD

**Issue by Lesbinary** [paracrawl/Malign#6]
I noticed that some resources in Malign repository are duplicated and not linked to the original upstream repository: 

- https://github.com/paracrawl/Malign/tree/master/utils
- https://github.com/paracrawl/Malign/tree/master/modules/moses_scripts

These `nonbreaking_prefixes` and `split-sentences.perl` can be replaced with:

- https://github.com/moses-smt/mosesdecoder/blob/8c5eaa1a122236bbf927bde4ec610906fea599e6/scripts/ems/support/split-sentences.perl
- https://github.com/moses-smt/mosesdecoder/blob/8c5eaa1a122236bbf927bde4ec610906fea599e6/scripts/ems/support/split-sentences.perl

The `nonbreaking_prefixes` are already isolated in a Bitextor repository: https://github.com/bitextor/nonbreaking_prefixes.

And these resources should not be duplicated in the same Malign repository, it is really confusing and difficult to maintain.

**Comment by kpu** [comment]
Nobody wants to pull in all of Moses though that's where these are maintained upstream.  At one point Marcin had a fit and made a repo with just the preprocessing scripts.  I note they're also in github.com/kpu/preprocess/.

**Comment by Lesbinary** [comment]
That repository is perfect, also for Bitextor. But still, the duplicated references in Malign should be revised.

**Comment by hieuhoang** [comment]
Leo has a point. Malign now has 2 versions of the moses script 
  modules/moses_scripts
  modules/bleualign-cpp/third_party/preprocess/moses
I'll make a decision when I take over

**Comment by kpu** [comment]
Not to mention the NLTK tokenizer in places. . . https://github.com/paracrawl/Malign/blob/master/scripts/tokenise.py

**Comment by kpu** [comment]
@hieuhoang You have the authority and ability to fix this now.

**Comment by bhaddow** [comment]
I agree, it's only going to get worse. @hieuhoang, how about a separate moses-scripts repo? (ie the bits of moses that people still use)

**Comment by hieuhoang** [comment]
I defer to marek for now. To many hands spoil the broth

**Comment by bhaddow** [comment]
What I meant was to remove the scripts subdir from mosesdecoder, and then everyone (including mosesdecoder) can load scripts as a submodule. @hieuhoang, this is your broth.

**Comment by hieuhoang** [comment]
Oh, fiddling with moses wasn't what I was thinking. It would set of a whole bunch of of emails of the making list. I would either use moses directly or kpu/preprocess and delete duplicates asap

**Comment by hieuhoang** [comment]
i've deleted the multiple copies of moses_scripts and non-breaking prefixes. The scripts in Malign will now take in an argument to the full moses repository.
   https://github.com/paracrawl/Malign/commit/300cff4816c5f5eca0c3cd8cea5275fe2de1ecea
   https://github.com/paracrawl/Malign/commit/663236e1b916d8eca4aea54e2a9ec43476aeb734

The little extra we spend on downloading & using the original code, the more we save on headache tablets

Sorting out tokenizer hairball soon
